cmd/kubeadm/app/cmd/completion_test.go

```diff
@@ -39,7 +39,7 @@ func TestNewCmdCompletion(t *testing.T) {
 	cmd := newCmdCompletion(&out, "")
 	parentCmd.AddCommand(cmd)
 	if err := parentCmd.Execute(); err != nil {
-		t.Errorf("Cannot exectute newCmdCompletion: %v", err)
+		t.Errorf("cannot exectute newCmdCompletion: %v", err)
```

Comment:
test.T log messages can be capitalized and golint does not complain about it.
Error messages on the other hand must be lowercase.

the cmd/kubeadm changes in this PR are not needed.